### PR TITLE
Add llama2 notebook tutorial

### PR DESCRIPTION
**Pull request by xiaoyu-work**
## Describe your changes

Add llama2 notebook tutorial

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Comment by guotuofeng** [comment]
Please fix the typos detected by the spell checker

**Comment by xiaoyu-work** [comment]
> Please fix the typos detected by the spell checker

Fixed.

**Comment by trajepl** [comment]
It got a little hard to review the ipynb file changes.  I noticed that most of content in jupyter note is about config files. Also not many runtime logs are showed to user.  Do you mind to convert it to markdown file?

File: examples/notebooks/llama2/notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,307 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Olive tutorial Llama2\n",
```
Comment (trajepl) [inline examples/notebooks/llama2/notebook.ipynb:7]: Olive Tutorial Llama2 on Azure Machine Learning?

File: examples/notebooks/llama2/config.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
+{
+    "azureml_client": {
+        "subscription_id": "3905431d-c062-4c17-8fd9-c51f89f334c4",
```
Comment (trajepl) [inline examples/notebooks/llama2/config.json:3]: Should we hide these kind of info?

Comment (xiaoyu-work) [reply examples/notebooks/llama2/config.json:3]: Oops I'll remove that.

File: examples/notebooks/llama2/notebook.ipynb (hunk 1 above)
Comment (xiaoyu-work) [reply examples/notebooks/llama2/notebook.ipynb:7]: Looks good to me.

**Comment by xiaoyu-work** [comment]
> Check out this pull request on  [ReviewNB](https://app.reviewnb.com/microsoft/Olive/pull/798)
> 
> See visual diffs & provide feedback on Jupyter Notebooks.
> 
> Powered by [ReviewNB](https://www.reviewnb.com/?utm_source=gh)

@trajepl please check this link to review notebook.

**Comment by guotuofeng** [comment]
> It got a little hard to review the ipynb file changes. I noticed that most of content in jupyter note is about config files. Also not many runtime logs are showed to user. Do you mind to convert it to markdown file?

you can try https://github.com/microsoft/Olive/blob/488340f73648d5d8ea482107facafa47f355b034/examples/notebooks/llama2/notebook.ipynb

**Comment by trajepl** [comment]
> https://github.com/microsoft/Olive/blob/488340f73648d5d8ea482107facafa47f355b034/examples/notebooks/llama2/notebook.ipynb

Yeah, I knew this. But it is hard to add comment to right place. ReviewNB looks great to me. 

Curious that is the ReviewNB suggested by copilot or something else?

**Review by trajepl — APPROVED** [review]

**Review by devang-ml — APPROVED** [review]

**Pull request merged**